### PR TITLE
Bug 1446236 - search-google.html.tmpl: Calculate sitesearch from urlbase

### PR DESCRIPTION
**Pull request by CyberShadow**
Remove a hard-coded BMO reference.

### CODE REVIEW

File: template/en/default/search/search-google.html.tmpl [hunk 1]
```diff
@@ -32,7 +32,7 @@
 <p>
 
 <form method="get" action="https://www.google.com/search">
-<input type="hidden" name="sitesearch" value="bugzilla.mozilla.org">
+<input type="hidden" name="sitesearch" value="[% urlbase.match('^([^:]*://)?(.*?)/?$').1 %]">
```
Comment (dylanwh) [inline template/en/default/search/search-google.html.tmpl]: With the stuff in #25 you can just write [% urlbase.host FILTER html %]

Comment (CyberShadow) [reply template/en/default/search/search-google.html.tmpl]: What about the case when Bugzilla is not installed at the site root?

Comment (dylanwh) [reply template/en/default/search/search-google.html.tmpl]: Aha! Perhaps... urlbase.host _ urlbase.path?
I'm not sure if sitesearch takes  path though.

Comment (CyberShadow) [reply template/en/default/search/search-google.html.tmpl]: Fairly sure it does, we've been using it for https://dlang.org/phobos/ for a while. Trailing slash is optional, not sure if it should be removed (for aesthetic reasons & to preserve behaviour identical to before the change)?

**Pull request merged**